server/api/src/rsBulkDataObjPut.cpp

```diff
@@ -600,7 +600,7 @@ bulkProcAndRegSubfile( rsComm_t *rsComm, const char *_resc_name, const std::stri
         }
         else {
             if ( ( flags & VERIFY_CHKSUM_FLAG ) != 0 && myChksum != NULL ) {
-                char chksumStr[NAME_LEN];
+                char chksumStr[LONG_CHKSUM_LEN];
```

Comment:
`LONG_CHKSUM_LEN`? Shouldn't this be `CHKSUM_LEN`?

Reply:
Agreed.

I just noticed that the comparison of checksums is performed twice. Once at this scope and another time in the call to `verifyChksumLocFile`.